### PR TITLE
Ankit | Test | Support GoCard less bank integration in other listed countries as well

### PR DESCRIPTION
**Pull request by ankitdubeyoffice**
https://app.clickup.com/t/86cwgv8qk

### CODE REVIEW

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.ts [hunk 1]
```diff
@@ -307,13 +307,6 @@ export class SettingIntegrationComponent implements OnInit, AfterViewInit {
             }
         });
 
-        this.store.pipe(select(s => s.session.currentCompanyCurrency), takeUntil(this.destroyed$)).subscribe(res => {
-            if (res) {
-                //This is not using no due to gocardless
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.ts:312]: revert this code only remove for only gocardless

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.ts [hunk 2]
```diff
@@ -362,6 +356,8 @@ export class SettingIntegrationComponent implements OnInit, AfterViewInit {
             }
         });
 
+
+
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.ts]: remove extra space

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.ts (hunk 1 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/settings/integration/setting.integration.component.ts:312]: Done

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.ts (hunk 2 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/settings/integration/setting.integration.component.ts]: Done

**Review by dilpreetsinghofficial — APPROVED** [review]

**Pull request merged**